### ISSUE TITLE
Support for Python3 + latest element tree

### ISSUE THREAD

**Issue by ksclarke** [joh/xmldiffs#10]
The newest Ubuntu has python3 and a newer version of element tree which no longer supports `getchildren()`. I don't care about backwards compatibility, so my changes were:
```
diff --git a/setup.py b/setup.py
old mode 100644
new mode 100755
diff --git a/xmldiffs b/xmldiffs
index fc4db2b..6d783af 100755
--- a/xmldiffs
+++ b/xmldiffs
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 """
 Usage: {prog} [OPTION] FILE1 FILE2
 
@@ -35,7 +35,7 @@ def indent(s, level):
     return "  " * level + s
 
 def write_sorted(stream, node, level=0):
-    children = node.getchildren()
+    children = list(node)
     text = (node.text or "").strip()
     tail = (node.tail or "").strip()
```
You'd probably want to add support for the older element tree + the newer version.

**Comment by michaelperrin** [comment]
Thanks @ksclarke, works like a charm 🙂

**Comment by joh** [comment]
This has now been fixed in #11.
